### PR TITLE
Sdk redux bookmarks center

### PR DESCRIPTION
**Pull request by CarlyLanglois**
Bookmarks component uses setView action to move map to selected bookmark.
User-defined bookmarks must each have a center and resolution.

### CODE REVIEW

File: __tests__/components/Bookmarks.test.js [hunk 1]
```diff
@@ -1,112 +1,100 @@
...
+import {expect} from 'chai';
+import {setView} from '../../src/actions/MapActions';
+import 'phantomjs-polyfill-object-assign';
+import ol from 'openlayers';
 import intl from '../mock-i18n';
-import {Bookmarks} from '../../src/components/Bookmarks';
+import Bookmarks from '../../src/components/Bookmarks';
+import BoundlessSdk from '../../src/components/BoundlessSdk';
+import configureStore from '../../src/stores/Store';
+import polyfills from '../polyfills'; // eslint-disable-line no-unused-vars
```
Comment (theduckylittle) [inline __tests__/components/Bookmarks.test.js]: @bartvde  has suggested changing this to "import '../polyfills" which removes the need for silencing the linter.  There should be some examples of that but I'm not sure if they're only in pull requests.  It's a small change but I'd rather we do it now than later.

File: __tests__/components/Rotate.test.js [hunk 2]
```diff
@@ -15,8 +15,6 @@ import configureStore from '../../src/stores/Store';
 
 import polyfills from '../polyfills'; // eslint-disable-line no-unused-vars
```
Comment (theduckylittle) [inline __tests__/components/Rotate.test.js]: Same comment as above.

File: src/actions/MapActions.js [hunk 3]
```diff
@@ -25,3 +25,9 @@ export function setView(view) {
     view
   }
 }
+export function extentToResolution(map) {
```
Comment (theduckylittle) [inline src/actions/MapActions.js]: I see this extentToResolution in the action-related files and in the MapConfigService.

1. There really is no need for a ConfigService with redux.  The configuration is the state.
2. I'm not seeing this used anywhere, so can it be eliminated?

Comment (CarlyLanglois) [inline src/actions/MapActions.js]: yeah it definitely can

**Pull request merged**